app/components/EventItem/index.js

```diff
@@ -18,6 +18,8 @@ type AttendanceProps = {
   event: Event,
 };
 
+export type EventStyle = 'default' | 'extra-compact';
```

Comment:
I think I'd prefer a `compact` boolean prop

Reply:
The reason why I'm not using a boolean here is because I'm working on another style: `compact`,
so I'm aiming to have atleast 3 styles with ` 'default' | 'compact'| 'extra-compact'`;

Should I just "pit stop" with a `compact` boolean here then?